core/DataAccess/Model.php

```diff
@@ -847,9 +847,10 @@ public function hasChildArchivesInPeriod($idSite, Period $period)
         while ($date->isEarlier($period->getDateEnd()->addPeriod(1, 'month'))) {
             $archiveTable = ArchiveTableCreator::getNumericTable($date);
 
+            $usableDoneFlags = [ArchiveWriter::DONE_OK, ArchiveWriter::DONE_INVALIDATED, ArchiveWriter::DONE_PARTIAL, ArchiveWriter::DONE_OK_TEMPORARY];
```

Comment:
Using partial or temporary archives kind of makes sense for me, but why are we also using invalidated archives here?
Shouldn't the archiving be triggered for invalidated archives, that can be used then?

Reply:
@sgiehl they should normally be rearchived before this archive gets launched (ie, weeks before months), but in the specific case of the GoogleAnalyticsImporter they don't have to be, since the plugin does not allow rearchiving of days w/ imported data (to make sure the imported data doesn't disappear through user or programming error). So in this case the days that are invalidated get ignored. It might be a better solution in the future to disallow invalidating them entirely...

Reply:
Might be good to add a small comment above the line then, so it's clear why we also use invalidated archives here. Just to prevent we remove it later by accident.